homeassistant/components/lcn/sensor.py

```diff
@@ -32,6 +36,17 @@
 from .entity import LcnEntity
 from .helpers import InputType
 
+SENSOR_DEVICE_CLASS = {
```

Comment:
I would recomment using a different name for this dict, to avoid confusion with HA constants
```suggestion
DEVICE_CLASS_MAPPING = {
```

Reply:
Done.